homeassistant/components/apple_tv/browse_media.py

```diff
@@ -1,43 +1,39 @@
 """Support for media browsing."""
+from typing import Any
 
 from homeassistant.components.media_player import BrowseMedia
-from homeassistant.components.media_player.const import (
-    MEDIA_CLASS_APP,
-    MEDIA_CLASS_DIRECTORY,
-    MEDIA_TYPE_APP,
-    MEDIA_TYPE_APPS,
-)
+from homeassistant.components.media_player.const import MediaClass, MediaType
```

Comment:
See https://github.com/home-assistant/core/pull/77946#pullrequestreview-1099871183

Reply:
Do we want to apply this just to `const` or should we also apply it to this line?
```python
from homeassistant.components.media_player.browse_media import (
    async_process_play_media_url,
)
```

Reply:
`async_process_play_media_url` is already imported in `media_player.__init__`, so it would make sense to do so.
https://github.com/home-assistant/core/blob/f87e873275305184585634f11e6256a72d6f40e1/homeassistant/components/media_player/__init__.py#L68

Reply:
done